test/test_entrypoints.py

```diff
@@ -1,38 +1,44 @@
-def test_raider__main__(script_runner):
-    ret = script_runner.run('RAiDER', '-h')
-    assert ret.success
+# Disabling for now as there doesn't seem to be a main runner
```

Comment:
I see now. We need to add a `main` to the appropriate module for the main RAiDER program. I'll address this in mine.